api/src/features/components/component_feature.py

```diff
@@ -0,0 +1,12 @@
+from typing import Dict
+
+from fastapi import APIRouter
+
+from common.components import COMPONENT_IDS
+
+router = APIRouter(tags=["component"])
+
+
```

Comment:
Not sure how this could fail, but should use the @create_response decorator anyway...

Reply:
Oops. WIll add.